pkg/providers/onepasswordconnect/onepasswordconnect.go

```diff
@@ -0,0 +1,90 @@
+package onepasswordconnect
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/1Password/connect-sdk-go/connect"
+	"gopkg.in/yaml.v3"
+
+	"github.com/helmfile/vals/pkg/api"
+)
+
+type provider struct {
+	client connect.Client
+}
+
+// New creates a new 1Password Connect provider
+func New(cfg api.StaticConfig) *provider {
+	p := &provider{}
+
+	return p
+}
+
+// Get secret string from 1Password Connect
+func (p *provider) GetString(key string) (string, error) {
+	var err error
+
+	splits := strings.Split(key, "/")
+	if len(splits) < 2 {
+		return "", fmt.Errorf("invalid URI: %v", errors.New("vault or item missing"))
```

Comment:
vault?

Reply:
1password is organized in vaults -> items -> [sections ->] fields
See also description of the vals-URI for this provider in the README